Cannot find module 'file-uri-to-path'

On linux, Ubuntu 16.04 latest master, also persists if I install file-uri-to-path manually and then build

```
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'file-uri-to-path'
    at Module._resolveFilename (module.js:440:15)
    at Function.Module._resolveFilename (/home/jpka/bin/hyperterm/dist/linux/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/jpka/bin/hyperterm/dist/linux/resources/app/index.js:7:23)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
```

--- a/app/package.json
+++ b/app/package.json
@@ -15,6 +15,7 @@
     "default-shell": "1.0.1",
     "electron-config": "0.2.1",
     "electron-is-dev": "0.1.1",
+    "file-uri-to-path": "0.0.2",
     "gaze": "1.1.0",
     "mkdirp": "0.5.1",
     "ms": "0.7.1",

--- a/package.json
+++ b/package.json
@@ -14,7 +14,6 @@
     "aphrodite-simple": "0.4.1",
     "color": "0.11.3",
     "electron-mocha": "^3.0.0",
-    "file-uri-to-path": "0.0.2",
     "hterm-umdjs": "1.1.3",
     "json-loader": "0.5.4",
     "mocha": "^3.0.0",

